charts/fission-all/templates/deployment.yaml

```diff
@@ -236,6 +236,14 @@ spec:
           value: "{{ .Values.pullPolicy }}"
         - name: ENABLE_ISTIO
           value: "{{ .Values.enableIstio }}"
+        - name: FETCHER_RESOURCE_REQUESTS_CPU
+          value: "10m"
```

Comment:
@xiekeyang Thanks for this. Instead of hardcoding them in the deployment YAML file, can you pull them from Helm Values, similar to other values are being fetched?